RecoLocalMuon/Configuration/python/RecoLocalMuon_EventContent_cff.py

```diff
@@ -39,3 +39,8 @@ def _updateOutput( era, outputPSets, commands):
 _updateOutput( run2_GEM_2017, _outputs, ['keep *_gemRecHits_*_*', 'keep *_gemSegments_*_*'] )
 _updateOutput( run3_GEM, _outputs, ['keep *_gemRecHits_*_*', 'keep *_gemSegments_*_*'] )
 _updateOutput(phase2_muon, _outputs, ['keep *_me0RecHits_*_*', 'keep *_me0Segments_*_*'])
+
+# RPC New Readout Validation
+from Configuration.Eras.Modifier_stage2L1Trigger_cff import stage2L1Trigger
+_updateOutput(stage2L1Trigger, _outputs, ['keep *_rpcNewRecHits_*_*'])
```

Comment:
Here you are keeping ```*_rpcNewRecHits_*_*``` to all ```[RecoLocalMuonFEVT, RecoLocalMuonRECO, RecoLocalMuonAOD]``` event contents.

Wasn't the conclusion we reached at the reco meeting saying that you don't need to save those products, but simply feeding them to the DQM so that you can monitor them? (In particular in view of a possible backport to 10_2)

Please clarify

Reply:
The former commit was done before the meeting. This new one incorporates what was discussed there.